src/datomish/exec.cljc

```diff
@@ -0,0 +1,67 @@
+(ns datomish.exec
+  #?(:cljs
+     (:require-macros
+      [datomish.pair-chan :refer [go-pair <?]]
+      [cljs.core.async.macros :refer [go]]))
+  (:require
+     [datomish.sqlite :as s]
+     [datomish.sqlite-schema :as ss]
+     [datomish.query :as dq]
+     #?@(:clj
+           [[datomish.jdbc-sqlite :as j]
```

Comment:
With the newest factory work, you should be able to drop the `:as j` here and below, and use `s/<sqlite-connection` in both situations.  Try it!